sycl/include/CL/sycl/stl.hpp

```diff
@@ -22,34 +22,52 @@
 __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 
```

Comment:
Temporary put this PR on hold to wait for more elaborate testing results.

Reply:
@romanovvlad, any updates on testing results?

Reply:
I put this comment to mark that PR cannot be committed yet in agreement with Slava.
@v-klochkov  Could you please help answering?